### PR TITLE
Submission

### PR DESCRIPTION
**Pull request by powerhouseofthecell**
Thank you!

### CODE REVIEW

File: asst1/asst1.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
+from collections import Counter
```
Comment (gcmerz) [inline asst1/asst1.py:1]: Aight just gotta say absolutely LOVE your github name but v unclear as to who you are

File: asst1/asst1.py [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+def swapchars(string_thing):
+    # initialize a counter and count each letter's occurrence
+    count = Counter()
```
Comment (gcmerz) [inline asst1/asst1.py:5]: You can acutally just call Counter() directly on the string! See the docs for details but you don't actually need the for loop underneath

File: asst1/asst1.py (hunk 1 above)
Comment (powerhouseofthecell) [reply asst1/asst1.py:1]: Sorry! I'm Nick Wong. My bad!

File: asst1/asst1.py (hunk 2 above)
Comment (powerhouseofthecell) [reply asst1/asst1.py:5]: I think that my goal in this case was being able to filter out non-alphabetic characters, though if there is a better way to do so, my bad!

Comment (gcmerz) [reply asst1/asst1.py:5]: There are lots of ways to do this! Don't worry, it's one of the hard parts about coding in python -- there are always a million ways to do the same task. We'll go over some other methods at the meeting tomorrow (:

File: asst1/asst1.py (hunk 1 above)
Comment (gcmerz) [reply asst1/asst1.py:1]: Hahaha don't be sorry it's all good

**Comment by powerhouseofthecell** [comment]
Thanks guys! See you tomorrow!

**Pull request closed without merging**